tests/augmentation/test_container.py

```diff
@@ -516,6 +516,40 @@ def test_class(self, device, dtype):
         _, out_labels = aug(img, labels)
         assert labels is out_labels
 
+    @pytest.mark.parametrize("B", [None, 1, 3, 5])
+    @pytest.mark.parametrize("C_i", [1, 3])
+    @pytest.mark.parametrize("C_m", [None, 1])
+    @pytest.mark.parametrize("keepdim", [True, False])
+    def test_masks_without_channel_dim(self, device, dtype, B, C_i, C_m, keepdim):
+        img_shape = (B, C_i, 1000, 500) if B else (C_i, 1000, 500)
+        mask_shape = tuple(x for x in (B, C_m, *img_shape[-2:]) if x)
```

Comment:
@shijianjian 

The idea here is to make it work with masks `B, H, W` or `H, W` and images `B, C, H, W`